hass_nabucasa/__init__.py

```diff
@@ -556,7 +574,11 @@ async def _subscription_reconnection_handler(
                 _LOGGER.info("No longer logged in, stopping reconnection handler")
                 break
 
-            await self.auth.async_renew_access_token()
+            try:
+                await self.auth.async_renew_access_token()
+            except CloudError as err:
+                _LOGGER.debug("Could not renew access token (%s)", err)
+                continue
```

Comment:
Do we need to delete a repair issue if this fails?

Reply:
No, it happens when the loop breaks.